src/it/java/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/FinalizersNotUsedTest.java

```diff
@@ -22,10 +22,10 @@
 import org.junit.jupiter.api.Test;
 
 import com.google.checkstyle.test.base.AbstractGoogleModuleTestSupport;
-import com.puppycrawl.tools.checkstyle.api.Configuration;
-import com.puppycrawl.tools.checkstyle.checks.coding.NoFinalizerCheck;
 
-public class NoFinalizerTest extends AbstractGoogleModuleTestSupport {
+public class FinalizersNotUsedTest extends AbstractGoogleModuleTestSupport {
+
+    private static final String[] MODULE = {"NoFinalizer"};
```

Comment:
Rename it to MODULES as it is not just one module but an array of them

Reply:
done 

https://github.com/checkstyle/checkstyle/blob/928aa10b9df1a6464b2088adc4a166dd3fb2f625/src/it/java/com/google/checkstyle/test/chapter6programpractice/rule64finalizers/FinalizersNotUsedTest.java#L26-L28